linux_os/guide/system/auditing/auditd_configure_rules/audit_rules_suid_privilege_function/tests/more_space.pass.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+# packages = audit
+
+echo '  -a  always,exit  -F  arch=b32  -S  execve  -C  gid!=egid  -F  egid=0  -k  setgid  ' > /etc/audit/rules.d/privileged.rules
+echo '  -a  always,exit  -F  arch=b64  -S  execve  -C  gid!=egid  -F  egid=0  -F  key=setgid  ' >> /etc/audit/rules.d/privileged.rules
+echo '  -a  always,exit  -F  arch=b32  -S  execve  -C  uid!=euid  -F  euid=0  -F  key=setuid  ' >> /etc/audit/rules.d/privileged.rules
+echo '  -a  always,exit  -F  arch=b64  -S  execve  -C  uid!=euid  -F  euid=0  -k  setuid  ' >> /etc/audit/rules.d/privileged.rules
```

Comment:
Would it be possible to incorporate some extra spaces in another "pass" test scenario, to reduce the number of scripts?

Reply:
I added them in other_key.pass.sh